docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -99,5 +78,4 @@ Get the master organization's username and password:
 juju run-action nms-magmalte/leader get-master-admin-credentials --wait
 ```
 
-Confirm successful deployment by visiting `https://host.nms.<your domain>` and logging in
-with the `admin-username` and `admin-password` outputted here.
+Confirm successful deployment by visiting `https://host.nms.<your domain>` and logging in with the `admin-username` and `admin-password` outputted here.
```

Comment:
I'm worried `host` isn't clear that refer's to the org's name?

Reply:
Fixed